lightning/src/blinded_path/payment.rs

```diff
@@ -246,8 +246,16 @@ impl BlindedPaymentPath {
 		Self { inner_path, payinfo }
 	}
 
-	#[cfg(any(test, fuzzing))]
-	pub fn from_raw(
+	/// Builds a new BlindedPaymentPath from its constituent parts.
+	///
+	/// Useful when reconstructing a blinded path from previously serialized components.
+	///
+	/// Parameters:
+	/// * `introduction_node_id`: The public key of the introduction node in the path.
+	/// * `blinding_point`: The public key used for blinding the path.
+	/// * `blinded_hops`: The encrypted routing information for each hop in the path.
+	/// * `payinfo`: The [`BlindedPayInfo`] for the blinded path.
+	pub fn from_blinded_path(
```

Comment:
IMO -- can we keep the previous name or call this `from_parts`?

Reply:
hmm I think `from_parts` is too broad. Maybe `from_blinded_path_and_pay_info` but could be too lengthly. I don't mind going back to `from_raw` but I wanted to keep it similar to `BlindedMessagePath::from_blinded_path`

Reply:
Ah okay, I didn't notice `BlindedMessagePath` already exposes a similar method, thanks! I'm good with the original name or this new one then.